index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Maria S. Resume</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<base target="blank">
+</head>
+<body>
+
+<header>
+		<h1>Maria S.</h1>
+		<nav>
+			<h2><a href="index.html">Resume</a> |
+			<a href="interests.html">Interests</a></h2>
+		</nav>
+	</header>
+
+<br>
+
+<hr>
+
+<h2>Education</h2>
+
+<p><b>[2004-2008] University of Pennsylvania & The Wharton School </b><br>
+<i>Joint degree - Huntsman Program in International Studies and Business </i> </p>
+
+<p><b>[In 2014] Urban Yoga Spa </b><br>
+<i> 200-hour yoga teacher training </i></p>
+	
+<p><b>[In 2015] General Assembly </b><br>
+<i> 10-week part-time Product Management course </i></p>
+	
+<p><b>[In 2017] General Assembly </b><br>
+<i> 10-week part-time Front-End Web Development course </i></p>
+
+<h2>Work Experience</h2>
+
+<p><b>[2013-Present] Zillow Group </b>- Corporate Development <br>
+<i> M&A, investments, and strategic partnerships for $ZG </i></p> 
+	
+<p><b>[2011-2012] Goldman Sachs</b> - Equity Research <br>
+<i>Internet </i></p>
+
+<p><b>[2010-2011] Canaccord Genuity</b> - Equity Research <br>
+<i>Internet </i></p>
+	
+<p><b>[2009-2010] FBR Capital Markets & Co</b> - Equity Research <br>
+<i>Internet and Video Game Software </i></p>
+
+<p><b>[2008-2009] BofA - Merrill Lynch</b> - Equity Research <br>
+<i>Leisure and Video Game Software </i></p>
+
+<h2>As Seen In</h2>
+
+<p>
+<b>10/05/16</b> MIPIM Startup Competition in NYC (<a href="http://www.mipim.com/startup-competition/">MIPIM</a>) 
+</p>
+
+<p>
+<b>08/23/16</b> HotPads launches utility cost estimates in partnership with UtilityScore (<a href="https://www.myutilityscore.com/">UtilityScore</a>) 
+</p>
+
+<p><b>06/22/16</b> Top 12 Women in Real Estate (<a href="http://www.metaprop.org/blog/2016/6/22/top-women-in-real-estate">MetaProp Blog</a>)
+</p>
+
+<p>
+<b>06/15/16</b> MTEC 2016 Conference in San Francisco (<a href="https://twitter.com/multifamilytech/status/743103659437760512">Twitter</a>) 
+</p>
+
+<p>
+<b>02/26/16</b> New York Demo Day Success (<a href="http://www.metaprop.org/blog/demo-day-2015-2016-new-york">MetaProp Blog</a>)
+</p>
+
+<p>
+<b>07/02/15</b> Elmspring Accelerator Partners with Zillow Group For Upcoming Class (<a href="http://www.prweb.com/releases/2015/07/prweb12823220.htm">PR Web</a> | <a href="http://elmspringaccelerator.com/big-announcement-zillow-collaboration/">Elmspring Blog</a>)
+</p>
+<br>
+<hr>
+
+<footer>
+		<nav>
+			<a href="https://www.linkedin.com/in/mariaseredina">LinkedIn</a> |
```

Comment:
The common convention for external links is to open them up in a new tab/window.  This is where an explicit use of the `target` attribute would be appropriate.  🔨